components/Card/card.css

```diff
@@ -0,0 +1,164 @@
+/* CARD COMPONENT */
+
+.card {
+    height: max-content;
+    width: 300px;
+    background-color: white;
+    border-radius: 2px;
+    box-shadow: rgba(50, 50, 93, 0.25) 0px 2px 5px -1px, rgba(0, 0, 0, 0.3) 0px 1px 3px -1px;
+}
+
+.card-head {
+    padding: 10px 0;
+    display: flex;
+    align-items: center;
+    text-transform: capitalize;
+}
+
+.card-image {
+    border-radius: 2px 2px 0 0;
+    width: 100%;
+    height: auto;
+    background-repeat: no-repeat;
+    background-size: cover;
+}
+
+.card-title {
+    font-size: 1.5rem;
+    font-weight: 500;
+    padding: .5rem 1rem 0;
+}
+
+.card-text {
+    font-size: .9rem;
+    color: var(--black);
+    padding: 1rem;
+}
+
+
+.ecomm {
+    width: 230px;
+    box-shadow: none;
+    cursor: pointer;
+    transition: all .2s ease-in-out;
+    position: relative;
+}
+
+.card-badge {
+    position: absolute;
+    top: 20px;
+    left: -1px;
+    background-color: var(--primary-color);
+    color: white;
+    font-weight: 600;
+    font-size: .8rem;
+    padding: .2rem .5rem;
+    transition: all .2s ease-in-out;
+}
+
+/* .ecomm:hover{
+    box-shadow: rgba(0, 0, 0, 0.24) 0px 3px 8px;
+} */
+
+.ecomm:hover>.card-badge {
+    background-color: var(--primary-hover);
+}
+
+.product-image {
+    position: relative;
+    text-align: center;
+    height: 280px;
+    overflow: hidden;
+}
+
+.product-image img {
+    width: 100%;
+    height: auto;
+    object-fit: cover;
+    object-position: bottom;
+}
+
+.ico {
+    cursor: pointer;
+    position: absolute;
+    top: 15px;
+    right: 10px;
+    background-color: transparent;
+    border: none;
+    outline: none;
+}
+
+.ico svg {
+    width: 20px;
+    transition: all .3s ease-in-out;
+
+}
+
+.ico svg:hover {
+    fill: var(--danger);
+}
+
+.ecomm .card-title {
+    font-size: .8rem;
+    font-weight: 400;
+}
+
+.ecomm .card-text {
+    padding: .4rem 1rem;
+    font-size: .9rem;
+    font-weight: 500;
+    color: var(--black);
+    white-space: nowrap;
+    overflow: hidden;
+}
+
+.ecomm .card-price {
+    padding: 0 1rem;
+    font-size: .8rem;
+}
+
+.price-now {
+    font-weight: 400;
+    padding-right: 5px;
```

Comment:
I think instead of pixels, rem unit can be used. The unit used should be consistent.